package/Dockerfile

```diff
@@ -13,7 +13,7 @@ WORKDIR /var/lib/rancher
 
 ARG ARCH=amd64
 ARG IMAGE_REPO=rancher
-ARG SYSTEM_CHART_DEFAULT_BRANCH=release-v2.5
+ARG SYSTEM_CHART_DEFAULT_BRANCH=dev-v2.5
```

Comment:
This needs to be done in a few other places like scripts/package, maybe another place.